src/ontology/Makefile

```diff
@@ -147,6 +147,9 @@ $(ROBOT_PLUGINS_DIRECTORY)/%.jar:
 
 # Specific rules for supplementary plugins defined in configuration
 
+$(ROBOT_PLUGINS_DIRECTORY)/uberon.jar:
+	curl -L -o $@ https://github.com/gouttegd/uberon-robot-plugin/releases/download/uberon-robot-plugin-0.2.0/uberon.jar
```

Comment:
By the way this makes me think that I should probably move that repository `uberon-robot-plugin` to the OBOphenotype organisation, since the plugin is completely Uberon-specific. Would you agree?

Reply:
I have no objections either way! Thank you!

Reply:
OK, not a priority, but I’ll do that at some point. It won’t change anything in practical terms but it will be “cleaner“.